code-collab-app/src/editorpage.js

```diff
@@ -85,6 +64,9 @@ class Editorpage extends React.Component {
 
     // Reset cell last edited state
     this.props.peer_data.current_cell = null;
+
+    // unlock the cell
+    this.props.peer_data.update_cell_lock(lastCellEdited, UNLOCK_CELL);
```

Comment:
This should probably just get moved into the `broadcastCellUpdate()` function, since we always pair broadcasting updates and unlocking the cell. For example, the timer lock release event also calls the `broadcastCellUpdate()` function and will need to release the lock as well.

It should go broadcast update -> cell unlock

Reply:
I fixed it in the last commit.